src/gitea/issue/async_issue.py

```
@@ -0,0 +1,277 @@
+            index: The index of the issue.
+            **kwargs: Additional arguments for the request.
+
+        Returns:
+            A tuple containing the issue as a dictionary and the status code.
+
+        """
+        response = await self._get_issue(owner=owner, repository=repository, index=index, **kwargs)
+        data, status_code = await process_async_response(response)
+        return cast(dict[str, Any], data), status_code
+
+    async def _edit_issue(  # noqa: PLR0913
```

Ruff (0.14.13)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)